src/components/SecondaryPanes/SecondaryPanes.css

```diff
@@ -3,7 +3,6 @@
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
 .secondary-panes {
-  overflow: auto;
```

Comment:
That's a good option, but I wonder if `overflow: hidden;` wouldn't work better here. In case we add more panes in the SecondaryPanes and we forget to manage potential overflow for new panes (or overflow in accordion headers). With `overflow: visible`, we may end up with content that overflows to the right and creates a horizontal scrollbar on the whole window.

Reply:
Ok, I will try that :)

Reply:
So I added it as `overflow-x: hidden;` to prevent hiding the vertical overflow, which we actually want.